modules/order/src/Form/OrderTypeForm.php

```diff
@@ -6,6 +6,9 @@
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\Core\Entity\EntityTypeInterface;
 
+/**
+ * Form for order types.
```

Comment:
This change is not needed, and the comment doesn't match what we usually do ("Provides an order type form.").